bin/osg-notify

```diff
@@ -173,6 +191,15 @@ def main():
     with open(args.message, 'r') as fp:
         contents = fp.read()
 
+        # Replace smart quotes and em/en dashes
+        contents = replace_smart_quotes_and_dashes(contents)
+
+        # Check for non-ascii characters
+        if has_non_ascii_characters(contents) and args.force_non_ascii == False:
+            print("ERROR: message contains non-ascii characters.")
```

Comment:
consider sending such error messages to stderr with `print("...", file=sys.stderr)`

Reply:
done